data-mermaid/mermaid-webapp
# Add blank row to observations + refactor

[trello ticket](https://trello.com/c/Fkb9Kqe7/907-add-a-blank-first-row-in-observations): add empty row to observations to make it more user friendly

to test:
1. go to each collect record page
2. check to see that for each new page, an empty observation row exists

recent updates include
- address fishbelt obs table test conflict
- update `hasFishBeltObservations` to have a length greater than one to account for empty row
- add jest-preview for future debugging
